modules/system/database/migrations/2022_10_17_000028_Db_Jobs_Update_Payload_To_Longtext.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+use Winter\Storm\Database\Schema\Blueprint;
+
+return new class extends \Winter\Storm\Database\Updates\Migration
+{
+    public function up()
+    {
+        Schema::table($this->getTableName(), function (Blueprint $table) {
```

Comment:
This can be done in a single migration file, no need to split it into two separate files.